src/core/mac/link_raw.cpp

```diff
@@ -220,6 +221,50 @@ otError LinkRaw::SetMacKey(uint8_t        aKeyIdMode,
     return error;
 }
 
+uint32_t LinkRaw::GetMacFrameCounter(void)
+{
+    return mSubMac.GetMacFrameCounter();
+}
+
+uint32_t LinkRaw::GetStoredMacFrameCounter(void)
+{
+    return mSubMac.GetStoredMacFrameCounter();
+}
+
+otError LinkRaw::SetMacFrameCounter(uint32_t aMacFrameCounter, otLinkRawMacFcStore aCallback)
+{
+    otError error = OT_ERROR_NONE;
+
+    VerifyOrExit(IsEnabled(), error = OT_ERROR_INVALID_STATE);
+    mSubMac.SetMacFrameCounter(aMacFrameCounter);
+
+    mMacFcStoreCallback = aCallback;
```

Comment:
Both the funtion 'LinkRaw::SetMacFrameCounter'  and 'LinkRaw::SetStoredMacFrameCounterr' are using the same variable 'mMacFcStoreCallback' to record the callback function. What if these two functions are set with two different callback functions?